src/Persistence/RepositoryDecorator.php

```diff
@@ -173,23 +173,23 @@ public function random(array $criteria = []): object
      */
     public function randomSet(int $count, array $criteria = []): array
     {
-        if ($count < 1) {
+        if ($count < 1) { // @phpstan-ignore-line
```

Comment:
I think we can add [`treatPhpDocTypesAsCertain`](https://phpstan.org/config-reference#treatphpdoctypesascertain) to the phpstan config to avoid requiring these exceptions

Reply:
Oh, it is disabled. Not sure why this is required